store/tikv/lock_resolver.go

```diff
@@ -375,6 +381,7 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 
 func (lr *LockResolver) resolveLock(bo *Backoffer, l *Lock, status TxnStatus, cleanRegions map[RegionVerID]struct{}) error {
 	tikvLockResolverCountWithResolveLocks.Inc()
+	cleanWholeRegion := true
```

Comment:
`cleanWholeRegion := l.TxnSize >= BigTxnThreshold`, no need to update it in the loop

Reply:
We need to distinguish normal resolve and lite resolve in metrics.

Reply:
Good point.

Reply:
I prefer to set cleanWholeRegion=false in resolve_lock_lite related code block, this makes the logic more clear.

Reply:
em, I don't think this will make logic more clear. A variable that is never changed should be more clear.